products/memgraph/v0.15.0/reference_guide/other-features.md

```diff
@@ -159,3 +159,150 @@ to true is matched:
 MATCH (n)
 RETURN CASE WHEN n.height < 30 THEN "short" WHEN n.height > 300 THEN "tall" END
 ```
+
+### Inspecting queries
+
+Before a Cypher query is executed, it is converted into an internal form
+suitable for execution, known as a *plan*. A plan is a tree-like data structure
+describing a pipeline of operations which will be performed on the database in
+order to yield the results for a given query. Every node within a *plan* is
+known as a *logical operator* and describes a particular operation.
+
+Because a plan represents a pipeline, the logical operators are iteratively
+executed as data passes from one logical operator to the other. Every logical
+operator *pulls* data from the logical operator(s) preceeding it, processes it
+and passes it onto the logical operator next in the pipeline for further
+processing.
+
+Using the `EXPLAIN` operator, it is possible for the user to inspect the
+produced plan and gain insight into the execution of a query. Currently, the
+various logical operators aren't fully documented as their behavior is subject
+to change. However, the behavior of most of them can be deduced from their name.
+In the future, additional information might be added to the output of the
+`EXPLAIN` operator.
+
+As an example, let's inspect the plan produced for a simple query:
+
+```opencypher
+EXPLAIN MATCH (n) RETURN n
+```
+
+```
```

Comment:
Add plaintext after the ticks like so:

```plaintext
text text text
```

I'm not entirely sure whether this is mandatory, but I think that docs on the web might not be rendered properly without it. In either case, this is more expressive so I prefer to do it that way.